README.md

```diff
@@ -93,6 +93,8 @@ this functionality might prove useful.
       - [timestamp](#timestamp)
       - [toJSON](#tojson)
       - [toJSONPretty](#tojsonpretty)
+      - [toUnescapedJSON](#tounescapedjson)
```

Comment:
This documentation has moved while this has been waiting (sorry). If you have a chance and wouldn't mind @nagypeterjob, please consider updating this PR with the documentation moved into docs/configuration.md. If you don't have time just say so, I can make the update if necessary.

Thanks.

Reply:
It would be really nice of you if you could do the change @eikenb. Also, I am not sure how to resolve the conflicts.

Reply:
I'll be happy to handle the doc changes and I can fix the conflicts while I'm at it.